tabify/gatewayRanksTabify.js

```diff
@@ -17,7 +17,7 @@ export default class GatewayRanksTabify extends ESTabify {
             });
             const resultArray = [];
             for (const GatewayRank in resultByRank) {
-                resultArray.push({GatewayRank, NSGCount: resultByRank[GatewayRank]});
+                resultArray.push({GatewayRank: Number(GatewayRank), NSGCount: resultByRank[GatewayRank]});
```

Comment:
this will throw an exception if you have null or any other string here.

Reply:
but rank from ES response is always a number, do we need to handle exception cases?

Reply:
If you can guarantee that